Dockerfile

```diff
@@ -28,20 +42,31 @@ COPY in_container.mk /root/Makefile
 COPY flytekit.config /root/flytekit.config
 
 # Docker support
-RUN apt-get update && apt-get install --no-install-recommends -y \
-       apt-transport-https \
-       ca-certificates \
-       gnupg-agent \
-       gnupg2 \
-       software-properties-common
-RUN curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add -
-RUN apt-key fingerprint 0EBFCD88
-
-RUN add-apt-repository \
-       "deb [arch=amd64] https://download.docker.com/linux/ubuntu \
-       bionic \
-       stable"
-RUN apt-get update && apt-get install --no-install-recommends -y docker-ce=5:23.0.6-1~ubuntu.18.04~bionic docker-ce-cli=5:23.0.6-1~ubuntu.18.04~bionic containerd.io
+
+run <<'DOCKER'
```

Comment:
out of curiosity, what is the purpose of this change

Reply:
more modern dockerfile syntax + ability to cache a lot more effectively (see latch-pod-base)